Data/4_Data.csproj

```diff
@@ -8,6 +8,12 @@
   <ItemGroup>
     <Compile Remove="Migrations\20200419124410_HighlightBooleanRemoval.cs" />
     <Compile Remove="Migrations\20200419124410_HighlightBooleanRemoval.Designer.cs" />
+    <Compile Remove="Migrations\20200505225055_DefaultRoles.cs" />
```

Comment:
What happened here? I think it can be removed since the other migrations never got committed right? Same goes for the other removed migrations that were added in this commit.
```suggestion
```